env/asset_step.yml

```diff
@@ -298,11 +298,23 @@ engines:
         template_work: houdini_asset_work
         template_work_area: asset_work_area_houdini
         version_compare_ignore_fields: []
+      tk-multi-shotgunpanel:
+        location:
+          version: v1.2.3
+          type: app_store
+          name: tk-multi-shotgunpanel
+        action_mappings:
+          Task:
+          - filters: {}
```

Comment:
This is one of the cases when i feel the expanded yaml is less readable than the compact. In other places we have this format:

```
        action_mappings:
          Task:
          - actions: [assign_task, task_to_ip]
            filters: {}
        actions_hook: '{self}/tk-maya_actions.py'
```

(note also the inclusion of `task_to_ip`. I suggest changing houdini to use this form.